i18n/en.toml

```diff
@@ -69,3 +69,9 @@ other = "Show navigation"
 other = "Redirecting..."
 [redirect_manually]
 other = "Click here if you are not redirected"
+
+# Error pages
+[error_404_title]
+other = "404 - Page not found"
+[error_404_description]
+other = "Sorry, the page you were looking for is not found, go back to the <a href='/'>homepage</a>"
```

Comment:
Either

```
Sorry, the page you were looking for was not found, go back to the home page.
```

or

```
Oops! This page doesn't exist. Try going back to our home page.
```

the latter taken from our existing site:

https://cuelang.org/asdf

Reply:
I've taken the latter